app/views/patron_requests/new.html.erb

```diff
@@ -61,7 +61,7 @@
                   <tr>
                     <td class="align-middle">
                       <label>
-                        <%= f.check_box 'barcodes', value: item.barcode || item.id, class: 'form-check-input', disabled: not_requestable, data: { 'itemselector-target': 'items', 'itemselector-id-param': item.id, 'itemselector-available-param': item.available?, 'itemselector-label-param': item.callnumber, action: 'itemselector#change' }%>
+                        <%= f.check_box 'barcodes', { multiple: true, class: 'form-check-input', disabled: cannot?(:request, item), data: { 'itemselector-target': 'items', 'itemselector-id-param': item.id, 'itemselector-available-param': item.available?, 'itemselector-label-param': item.callnumber, action: 'itemselector#change' } }, item.barcode || item.id, "" %>
```

Comment:
not_requestable = cannot?(:request, item), is there a reason to not use the variable?

Reply:
That variable probably wasn't there when I first did the PR. Do you think the variable clarifies the intent, or was it introduced to avoid duplicate processing?

Reply:
duplicate processing, it is used for the label as well.